components/PlantCard/PlantCardStyles.module.css

```diff
@@ -0,0 +1,52 @@
+.Card {
+  height: 40vh;
+  width: 20vw;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  border-radius: 12px;
+  background-color: white;
+  box-shadow:
+    0 24px 38px 3px rgba(0, 0, 0, 0.14),
+    0 9px 46px 8px rgba(0, 0, 0, 0.12),
+    0 11px 15px -7px rgba(0, 0, 0, 0.2);
+}
```

Comment:
![image](https://github.com/user-attachments/assets/5e9c243c-b8fb-4989-afd9-4a6cbfcedd59)
The text goes outside of the div depending on the browser zoom, maybe consider setting an 'overflow' attribute? or setting a fixed font size for the text since it currently changes depending on the zoom